### DISCUSSION TITLE
Final Thinking Cap 💡

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/24SU-ASIAAM-191A#52] [Thinking Caps]
### Due Wednesday 7/31

Respond to the following questions: 
> ## What are your thoughts about the future of web development for civil society? 
> ## Which visiting voice left the most impact on you?
> ## What personal connections to people (if any) would you like to maintain from this class going forward?

## Submission
Share your thoughts in at least **4-5 sentences** before Wednesday, 7/31 and respond to **one** other student.

**Comment by stinest** [comment]
I believe web development is a fast-evolving work that has a lot of promise for civil society, now and especially in the future. After 6 weeks of learning the ins and outs of HTML, JavaScript, and CSS, I feel especially more hopeful for its potential to create powerful tools that can benefit communities of any size. As for the most impactful visiting voice, I mostly enjoyed and connected with Yoh Kawano's presentation. As a computational scientist for GIS and spatial DS _and_ traveler, his story was incredibly inspiring! Overall, this class was absolutely amazing and I am so happy that I took the opportunity to take it. I would love to keep in touch with everyone since this class would not have been the same without yall!!

**Reply by Keiya-Wada** [reply]
I agree with what you said! Yoh was also very impactful for me because it was inspiring to see a fellow Japanese person making positive impacts in this world. Although I have learned other coding languages like python, Java, etc, the languages we learned in this class was the first one I felt like had an actual impact on my community, so I also believe in the fast evolving potential of web development.

**Reply by NellieNguyen149** [reply]
Good point!! Web development is a quickly evolving field! While I face a lot of struggles with HTML, JavaScript, and CSS (ahha...), there is a limitless amount of opportunities to utilize these tools for civil society. I also totally agree with you, Yoh Kawano's presentation was super interesting to learn about, as all the visiting voices provided multiple types of ways to approach mapping, GIS, and overall uplifting community voices. I hope to stay in touch with you and everyone else too!! :D

**Comment by kchhoeu** [comment]
I believe that web development is an interesting tool for civil society, I think it has its pros and cons. After not knowing anything about web development prior to this class I think it has really good potential for helping civil society if used well and properly. I think that all the visiting voices were impressionable, and they all do amazing things for the community, but the visiting voice that I thought was the most impressionable on me was Yoh Kawano. I think it also had to do with the fact that I was in Japan while he talked so seeing the history and being in the country that his presentation and community of interest was in was really interesting, seeing how life can be so different depending on prefecture and how he directly interacted with the community was fun and insightful! I think that everyone in this class has been so nice, and they really made the class fun and worth waking up at 7AM for! I would love to keep in touch with everyone! Thank you for an amazing class!

**Reply by zhamic7** [reply]
I also liked Yoh's presentation a lot because of how involved he is with the communities he's worked with. It was especially inspiring to see someone combine skill sets in coding, GIS, and even film-making in their work. That's so cool that you were also in Japan during the time of his talk and that it brought you a unique perspective to his presentation!

**Reply by sjeyakum** [reply]
I agree, I think there is definitely a lot of potential, but it is important to consider the pros and cons of the tool depending on the purpose and community. I can't believe you had to wake up at 7AM for class, but I'm glad you enjoyed this course!

**Reply by sachigill** [reply]
I agree, these tools can either be viewed in a positive or negative light depending on how you use them and what goals you are trying to achieve, whether someone is actually trying to uplift a community or just doing something for the sake of profit or trying to be a savior. I hope that developers can start to go into their work with a proper mindset, and incorporate ethics and design justice into their decisions, so communities and their perspectives can be centered.

**Comment by Keiya-Wada** [comment]
I believe that the future of web development is a very bright one, allowing the voices of communities to be heard. Although it does have some disadvantages to it, since it is still a developing field, we should find better solutions to these problems over time. I'm sure that this six week class only covered the surface level coding and ethics of GIS, but I still feel very inspired to use these tools to create a positive impact on my local community. Out of all the visiting voices, I personally felt like Mariah's visiting voice was the most impactful for me. I think the deciding factor for me was that it was our first ever visiting voice in this class, so it opened up my eyes for the countless possibilities of GIS and maps in general. This is also a bit unrelated, but her voice was very calming and easy to follow along, so she was also a great presenter! Although I am not a UCLA student, I still felt very connected to this class, so if possible I would like to maintain my connection within people from this class.

**Reply by jasminereyz** [reply]
Hi Keiya! I was also most impacted by Mariah's visiting voice for the exact same reasons! I truly do feel that her being our first voice also opened up my eyes to a whole new world of mapmaking and web design that I did not even know existed. I also agree that her voice was very calming for me too. Even though you aren't a UCLA student you are always welcome <3!!

**Reply by kianaptran** [reply]
Hey Keiya, I had the same experience listening to Mariah's visiting voice! It is also so cool that you are only a rising high schooler, and being exposed to a course like this, something with so much social awareness and ethics, is such a great baseline for your coding career! It can become easy to have your morals compromised in any career if not taught ethics early on, so being aware of the power you can hold as a programmer is such a fantastic start.

**Reply by sjeyakum** [reply]
Sometimes it feels like we’ve covered so much in just six weeks, but I totally agree that there’s still so much more to learn about regarding critical cartography, GIS ethics, web development for civil society, etc. I also enjoyed Mariah's presentation the most and also appreciate how as the first visiting voice, she really got us thinking differently about maps and the bigger picture when it comes to mapping issues and ethics. And yes, her voice is calming :)

**Comment by zhamic7** [comment]
Considering how much of media has moved online in the form of webpages, I think web development will continue to critically influence how we obtain information and communicate ideas with each other. The visiting voice that was most impactful for me was Nina Kin. Her journey from LA County to LA Metro was informative for me because I am interested in the public sector for engineering but don't get to hear about relevant experiences as much. It was also inspiring to see how she manages work-life balance because I also have hobbies important to me that I want to make sure I don't sacrifice for work. I hope I can keep in touch with everyone here -- this class has had one of the friendliest and most supportive atmospheres! Thank you to everyone for making these six weeks so much fun!

**Reply by kianaptran** [reply]
Hey Michelle! I also loved learning about Nina's experience, especially her advice about being more selective about who you spend time with. One of my problems is being a chronic people-pleaser, but her advice on avoiding those who don't value you or your time was an excellent wake-up call to be more upfront and transparent with my boundaries. I also love looking at your web designs weekly for lab work, hehe!

**Reply by nesslee123** [reply]
Really great response! I also really enjoyed Nina Kin's visiting voice because I relate to the struggle of overworking myself. Similar to your experience, I also found it inspiring how she maintains a work-life balance because I think it's important to have a balanced lifestyle.

**Comment by jasminereyz** [comment]
I believe that that the future of web development is truly in the hands of us! Before taking this class, I never thought much about web development and how issues affect usage and accessibility. I think that there are definitely a brighter side to all of the terrors we've learned about, such as what our visiting voices brought. I felt most connected to Mariah's lecture on humanizing technology and I still think about her quote on how maps don't show the amount of deaths on a street which really stuck with me. I think her lecture was pivotal to my thinking on maps as it was very black & white before, but now even I want to create maps that have so much more meaning and I think that is exactly what our class calls for, to create maps for empowerment. Moving forward from this class, I would like to stay connected to my amazing partner Caitlynn as we really do love our final project and we hope to expand our project across the UC system!

**Reply by kianaptran** [reply]
Hey Jasmine! This is a funny story for the other classmates who don't already know: She is my roommate, and I low-key forced her to take this class with me. :D But I couldn't have had a better roomie to struggle with coding for the first time than you! We both thoroughly enjoyed working on the projects every week, even if it was always challenging (we are technologically illiterate).

**Reply by caitlynn-hauw** [reply]
I LOVE YOU JASMINE!!! YOU BOTH DID IT!

**Comment by kianaptran** [comment]
Going into this class, I was unaware of how intricate web development is. Although I knew there was already some form of manipulation from Capitalism and profit-hungry corporations, the extent I learned from this course was overwhelming. It is easy for an online consumer not to pay attention when companies target you. From the readings, especially the one on Design Justice, I am hopeful that the future of web development will become more accessible to the general public and cater to them instead of using the public to push a capitalist agenda. 
I thoroughly enjoyed all the visiting voices, as they provided diverse experiences and stories from their work with web development. The voice that I found most impactful was Mariah's. As she was the first visiting voice, I saw her lecture as highly digestible to someone new to GIS. She opened my eyes to what GIS _could_ be, whether physical or not, challenging my colonized perspective. Additionally, she brought awareness to how much power maps hold. As a future map maker, I became more conscious when creating our mapplications to reflect on the power and use it properly to serve the community best.
I am extremely happy to have made connections through this class, and I also learned so much from my partner during the mentor-mentee breakout room!

**Reply by caitlynn-hauw** [reply]
Hey Kiana! I believe in the context of Mariah's presentation, your use of "colonized perspective" is really compelling. As somebody born in America, I came into this class with the intention of solving problems, which I believe is part of the "colonized perspective"; a belief that our thinking is superior to the actual communities who are facing a particular problem. As you said, Mariah's presentation really caused me to question who we are empowering through upholding the status quo, and more often than not, those are systems in power.

**Comment by caitlynn-hauw** [comment]
If I'm being honest, the future of web development for civil society concerns me. I think technology in general often sells us a solution, which is merely a symptom of the capitalist society we live in. It is an industry currently incapable, as a system, of truly empowering civil society. I'm worried that although web developers may be equipped with great technical skills that are far beyond the scope of this class, they won't lead with the knowledge of ethics designed to empower communities, which can result in further oppression and marginalization. However, in spite of my fears, it is classes like these that make me hopeful. Although there might be greater forces systematically working to disempower communities, I've learned that we have the individual capability to empower local communities with web development. With the privileges associated with attending a school like UCLA, I believe we have a responsibility to do good. When we witness injustices, like, as Mariah described, "a map [being] a magician capable of a disappearing act," it is my hope that we continue to utilize web development to find and uplift silenced narratives. 

Mariah's visiting voice particularly stood out to me because there is an inherent, complex relationship with maps embedded into her Indigenous identity. I learned that by merely existing in the space of critical cartography, at a table not initially intended for her, she is actively working to dismantle the power structures maps uphold. I was also fascinated by the idea of contrived binaries that are created by those in power, which attempt to create stark right and wrong, subjective and objective, and black and white, in a very nuanced world. Through Mariah's voice, I learned that it's crucial we continually question who we are empowering by upholding the status quo, or we risk upholding systems of power and disempowering communities. I feel privileged that she consciously chose to translate her Diné introduction to us. One day, with the skills from this class and insight from her voice, I hope to be worthy of that privilege. 

Finally, I would like to maintain a relationship with everybody in this class, from the students to the visiting voices. The professor is a little strange, but maybe him too. I feel very grateful and privileged to have had the opportunity to take this class; it has truly changed my perspective on the world and my future aspirations in web development. Ever since I was a kid, I was fed an American-made savior complex to make me feel strong. However, through this class, I've realized that it is not my responsibility to save anyone. If I wish to help empower a community, they are more than capable of saving themself. Rather, it is my responsibility to utilize my privilege to help support a community in finding their own solutions to their own problems.

**Reply by giaari15** [reply]
That concern is so real and I agree that courses like this one gives me hope. I think what really helps me stay hopeful for the future is envisioning what tech and society looks like beyond capital gain. I think of digital archival sites with no paywalls and accessible information and individual portfolio sites which give a glimpse as to what web dev could look like when the main goal is not profit. Also upvote on that it is not our responsibility to save anyone!! Savior-complexes are... questionable a lot of ways and being a part of a community rather than "saving" one is much more valuable.

**Comment by sjeyakum** [comment]
When I think about the future of web development for civil society, the first word that comes to mind is "dynamic." In this sense, web development should constantly be evolving by offering numerous opportunities for enhanced public engagement through interactive features like forums and surveys. However, I believe this dynamism also brings its own set of challenges, such as the risk of designers creating biased representations that can distort public perception and hinder meaningful dialogue. As such, I think it is crucial that the future of web development is driven by collaboration between developers and community members to balance these opportunities and challenges, ensuring platforms reflect changes in technology and society while prioritizing community interests and needs. I think that's why out of all the visiting voices, Mariah's presentation left the most impact on me. Her introduction being a map itself really opened my eyes to how standardized and Westernized representations of maps have become, and how these displays can reveal or conceal information, influencing public discussions about maps. Her presentation made it clear that when designers are disconnected from the community, it can lead to a savior complex or result in significant misrepresentations. 

Also, I would love to stay connected with all of you! Next year will be my last year at UCLA, so I would be down to meet up or at least keep in touch somehow! I really enjoyed this class, and I'm glad I got to take it this summer with you all :)

**Reply by stinest** [reply]
Interesting response! Mariah's presentation was also super amazing and left me with a lot of insight into the representation of maps which helped a lot in assessing design justice. Your discussion posts and lab assignments have always been super cool to read and see, and I hope to see you on campus next year!!

**Comment by giaari15** [comment]
I think the future for web development and civil society has only one real direction to head towards which is for people by people. In our current capitalist and ultra-consumerist societal contexts, innovation in any sense is stunted when web dev and design is catered towards a profiteering agenda where advertisement takes the forefront. I believe accessibility, interactivity, and creative web dev and design is the future that centers human experience. The best example I can think of what it may look like in the future is a more developed peak-Tumblr era style webpages as art, information, webpages, writing, and more were personalized and attuned to the goal of the pages. 
The visiting voice that impacted me the most was Yoh Kawano, although every visiting voice was interesting to listen to. My own interests in film with a focus on community made me feel more connected to Yoh's experience with his film and being in Japan. I especially appreciated when he talked about the decision to insert or not insert himself in his own film and the thought process around that decision.
I would love to stay connected to everyone! This class was very fun to be a part of and made taking classes in the summer worthwhile. I am excited to see how everyone's final project ends up and what they will do with the knowledge we have gained from this course.

**Reply by kchhoeu** [reply]
Hi Gia! I thought your discussion post was really insightful and interesting! I agree that Yoh Kawano was such a interesting visiting voice, and I think it is cool you connected with the film aspect because of your own interests! All of your projects have been so interesting in this class I hope you have an amazing school year next year!

**Reply by chrissanrow** [reply]
GIA!! I agree with your assertions regarding the effect on capitalism and consumerism on web design when it comes to its application for civil society. Also, Tumblr era webpages are so fun and web design in general is such an interesting form of creative expression, when it's not being plagued by corporate marketing. we should both get on spacehey..

**Comment by nesslee123** [comment]
When it comes to the future of web development for civil society, I think it is very quickly technologically improving which is positive as long as we continue considering civil society, equally. This concept was discussed a lot during the quarter and I believe it is a continuously important idea that as we technologically advance, we must remember to keep considering civil society in these advancements. If we can keep doing this, I feel like there can only be positive changes and enhancements made to both web development and civil society. The visiting voice who impacted me the most was Nina Kin because I really resonated with her discussion of pushing yourself too hard and burning out. This is something I often struggle with academically and I thought Kin’s perspective was very insightful. I especially enjoyed her mention of not engaging in hobbies to relieve stress that actually cause more stress because I feel like this is something I struggle with often and I would like to start filtering my own hobbies and habits to reflect this idea. I would honestly be interested in staying connected to many people in this class because I feel like everyone added something unique to the class dynamic and were overall really nice and fun people to be around (virtually).

**Comment by chrissanrow** [comment]
I think that web development has (especially in recent years) become a more widely accessible and attainable skill for the common person. There are so many free resources online to support individuals trying to learn web development. I think that the issue of technodeterminism remains though, even in the presence of this self-empowered learning, where many individuals or small organizations seek to find technological solutions in pursuit of filling a "gap in the market". However, as more and more people begin to realize the necessity of ethics in computing, web development becomes a more critical and empowering tool for civil society.

As a queer/nonbinary person in computer science, Nina's visiting voice left the most impact on me. I have always found it difficult to relate to the ethical motivations of my peers in the field and have struggled with concerns of finding a place in the industry where I can still feel involved in civil society. I think her journey is really inspiring and motivates me to continue pursuing software engineering! I would love to stay connected with everyone in this class, even those who I didn't get a chance to speak to, along with the professor and the visiting voices. Thank you all for such an enjoyable and inspiring class (my favorite class at UCLA so far), everyone is so fun and talented!

**Comment by sachigill** [comment]
When thinking about the future of web development for civil society, I think it's crucial to center the people and communities involved, and ultimately make sure to prioritize the community needs over profits. I believe that web developers need to stop viewing everything through a neutral lens and consider all perspectives in their work so they can create inclusive platforms and use their skills to actually empower people, rather than continuing the oppression of already marginalized groups. The visiting voice I found most impactful was Mariah because I was exposed to a whole new perspective of what can even be considered a map and how narrow minded society can be regarding the concept of maps, as well as the different power dynamics or imbalances that maps can reveal or perpetuate. I think everyone in this class was very supportive and friendly, and I always enjoyed reading the jokes people would make in the zoom chats, and would love to stay connected with everyone!

**Comment by NellieNguyen149** [comment]
While there is still a lot of progress to be made regarding the current uses of web development for civil society (as we have discussed during some lectures already), I think I am optimistic about the future! Starting with this course, there is already a blossoming understanding of how to better utilize web development to truly provide a platform for community voices and society, instead of just trying to "solve issues" using technology alone. As someone who learned a great amount about this topic because of this course, I hope the same lessons I learn become more prevalent and we continue to recognize what types of approaches are more impactful and beneficial for the people. There are still existing barriers to using web development alone to improve civil society (a big one being barriers to access to technology), but it can play a role in a bigger, more inclusive movement toward uplifting society and civil movements. 

In relation to what I discussed, the visiting voice that left the biggest impact on me was Mariah because of the way she re-introduced the concept of mapping to us through showing how much influence the creator has on the map. I used to believe that it would be easy to recognize an "inaccurate" map because mapping is seemingly objective; how can you "lie" about physical locations? Isn't mapping just representing facts? However, I realized that mapping can suppress or underrepresent communities without the viewers even realizing it, which I connected to ideas I've learned from my statistics classes where graphs and charts can be manipulated and portray one idea, but it is actually another idea.  I appreciate Mariah's thorough explanations about how many maps can be misleading, a reflection of deeply rooted unfair power dynamics and the Westernization of our geographical understanding. 

This is one of the best classes I have ever taken, especially as a virtual class. I have a notoriously bad habit of being disconnected from online courses (and sometimes even my in-person classes... oops...), but I truly felt connected to the interactive nature of the course and the liveliness of my classmates. This has also been one of the most challenging courses I have ever taken, directly challenging the mindset I'm used to and showing me that there are many other (more impactful) ways to approach civil society and support the people instead of just "problem --> solution" based methods. I hope to stay in touch with everyone through the Discord server we share, and if anyone were to want help with whatever projects or ideas they have, I would love to be updated about them! Thank you to everyone for a wonderful class, and I wish you all success with your endeavors :D

**Comment by kimada62** [comment]
Web development will be a powerful tool for however people decide to use it. It can be an incredibly great tool for social change, or it could continue to distort a truth by negating certain information. I think Mariah's presentation really grounded me into the class. For some reason, I particularly remember her slide asking us which photo is the map. The answer was all. One of the photos depicted indigenous art (please correct me if this description is inaccurate) that was actually a carving of a map. It was not at all immediately obvious at first glance for people who have never seen this kind of mapping. However, it was an incredibly powerful way to say that mapping tells peoples' stories. It can continue to lift people and their culture if used with care. Like many others, I have a habit of overpromising to stay in contact- life just catches up and all the sudden it has been years. I hope to cross paths with anyone in this class. While I didn't get to interact with many of my classmates, I was truly inspired to see some of their coding skills and ambition to make change. Hope to see you all and your absolutely massive platforms for social change in the future!
